docs/services/paymentservice.md

```diff
@@ -8,33 +8,39 @@ processed.
 
 ## Initializing OpenTelemetry
 
-It is recommended to use a Node required module when starting your NodeJS
-application to initialize the SDK and auto-instrumentation. When initializing
-the OpenTelemetry NodeJS SDK, you optionally specify which auto-instrumentation
-libraries to leverage, or make use of the `getNodeAutoInstrumentations()`
-function which includes most popular frameworks. The `tracing.js` contains all
-code required to initialize the SDK and auto-instrumentation based on standard
-OpenTelemetry environment variables for OTLP export, resource attributes, and
-service name.
+It is recommended to use wrap your app using an OpenTelemetry initialization
```

Comment:
The wrap concept could be explained a bit more. For example, for me, it's not clear if we use wrapping in the code below.

Reply:
Tried to clarify the approach in [b626a9c](https://github.com/open-telemetry/opentelemetry-demo/pull/583/commits/b626a9c80d23b13a8b99be475e9093e4aa0c749b). :slightly_smiling_face: